AzureDevOps/CrossPlatformBuilder.yml

```diff
@@ -3,24 +3,24 @@ jobs:
     # Matrix strategy to run builds across different OS and MATLAB release combinations
     strategy:
       matrix:
-        linux-R2024b:
-          imageName: ubuntu-latest
-          release: R2024b
         linux-R2025a:
           imageName: ubuntu-latest
           release: R2025a
-        mac-R2024b:
-          imageName: macOS-latest
-          release: R2024b
+        linux-with-latest:
+          imageName: ubuntu-latest
+          release: latest
```

Comment:
To use the `latest` keyword, I've updated the identifier to `linux-with-latest` (as simply `linux-latest` could be misleading). Does anyone have a better name or is this fine?